.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+name: PR Title
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  test:
+    name: Check PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: deepakputhraya/action-pr-title@master
+        with:
+          regex: '^[a-z]+!?: .+$|^chore: release \d+\.\d+\.\d+$'
+          allowed_prefixes: "fix,refactor,feat,docs,chore,style,test" 
+          disallowed_prefixes: "feature,hotfix" 
+          prefix_case_sensitive: true 
```

Comment:
Generally a conventional commit message looks like:
```
<type>(<scope>)<!>: <message>
```
`<scope>` and `<!>` being optional.

IMHO a single regex should be sufficient, i.e. no need to provide extra `allowed_prefixes`, `disallowed_prefixes` and `prefix_case_sensitive`.

How about something like:
```
^(feat|fix|docs|style|refactor|perf|test|chore)(\([^\r\n]*\))?!?: .+$
```

This regex allows for the enumerated 'types' (though I have personally never used `perf` or `style` yet). 
It also allows for the 'scope' and takes into account breaking changes `!`.

It would match:
* `feat: Foo`
* `feat!: Bar`
* `feat(some): Baz`

Also it would match the  `pull-request-title-pattern` below.

Reply:
Maybe it would also make sense to extend above regex so that the first character of `<message>` is always a capital letter, e.g. `[A-Z].+$` instead of simply `.+$`.

Reason being that when the messages are included in the release notes it looks consistent.